### ISSUE TITLE
Type definitions

### ISSUE THREAD

**Issue by nhbravo** [hydropuzzle/react-native-unmount-animatable#1]
Hello! Could you please give us TypeScript definitions?

**Comment by nhbravo** [comment]
any updates on this?

**Comment by sobstel** [comment]
No, sorry, no plan to work on this any time soon. Pull requests are welcome though.

You could also try simple alternative using hooks: https://github.com/oblador/react-native-animatable/issues/132#issuecomment-786870602
